vitessce/config.py

```diff
@@ -1495,6 +1495,65 @@ def to_dict(self, base_url=None):
             "layout": [c.to_dict() for c in self.config["layout"]]
         }
 
+    def get_views(self):
+        """
+        Provides all the views in the config.layout object list
+
+        :returns: A list of VitessceConfigView objects.
+
+        """
+        return self.config["layout"]
+
+    def get_view_by(self, view_by):
```

Comment:
```suggestion
    def get_view_by_index(self, view_by):
```

I would split this into two functions, `get_view_by_index` and `get_first_view_by_type`

to make it more explicit the functionality